sdcm/nemesis.py

```diff
@@ -3622,6 +3619,7 @@ def data_validation_prints(args):
     def wrapper(*args, **kwargs):  # pylint: disable=too-many-statements
         # pylint: disable=too-many-locals
         # pylint: disable=too-many-branches
+        LOGGER.info("Args: %s", args)
```

Comment:
@juliayakovlev was this part of the change ?, if not  please remove

Reply:
removed